tests/test_memory.py

```diff
@@ -137,12 +137,10 @@ def __init__(self):
         self.addInterface(mc)
 
         # Add Device
-        modeConfig = ['RW','RW','RO','RO']
         for i in range(4):
             self.add(MemDev(
                 name       = f'MemDev[{i}]',
                 offset     = i*0x10000,
-                modeConfig = modeConfig[i],
```

Comment:
@slacrherbst You removed modeConfig, so now the test only covers "RW" registers instead of both "RW" and "RO". Shouldn't we still be testing the "RO" case in this script? If not, then wouldn’t it make more sense to remove the loop over 4 entirely and just instantiate MevDev once?

Reply:
It's not clear to me there is a clan way to test the RO registers with the limitations of the emulator. The best test relied on a bug in the code which allows RO registers to be written.